Makefile.dev.mk

```diff
@@ -48,7 +48,7 @@ dev/install/protoc-gen-go: ## Bootstrap: Install Protoc Go Plugin (protobuf Go g
 	go get -u github.com/golang/protobuf/protoc-gen-go@$(GOLANG_PROTOBUF_VERSION)
 
 dev/install/protoc-gen-validate: ## Bootstrap: Install Protoc Gen Validate Plugin (protobuf validation code generator)
-	go get -u github.com/envoyproxy/protoc-gen-validate@$(PROTOC_PGV_VERSION)
+	go get github.com/envoyproxy/protoc-gen-validate@$(PROTOC_PGV_VERSION)
```

Comment:
Why removing the `-u`?

Reply:
At first, we install `make dev/install/protoc-gen-go` with the provided version (make target right above).
Then we run `make dev/install/protoc-gen-validate` with `-u` and it updates `protoc-gen-go` to the latest version, which is incompatible for us. 

Maybe all dependencies should be installed without `-u` flag

Reply:
I see. Well removing `-u` everywhere makes sense to me. Let's stick with the specific versions we know are working.

Reply:
+1 to removing `-u`